NETPROGRAM/String_Exercise/Exercise01.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NETPROGRAM.String_Exercise
+{
+    public class Exercise01
+    {
+
+        public string SnakeCase(string s)
+        {
+            var snakeString = s.ToLower();
+            for(var i = s.Length - 2 ; i > 0 ; i--)
+            {
+
+                if (s.ToLower()[i] - s[i] > 0)
```

Comment:
Anh không thể kiếm được 1 cách nào đó dễ hơn sao??

```suggestion
                if (char.IsUpper s.ToLower(s[i]))
```

`s.ToLower()` được gọi trong suốt vòng lặp mặc dù kết quả nó không bao giờ thay đổi???

Việc kiểm tra viết hoa viết thường, sao anh không thể google 1 cái để tìm cách chính thống nhất

Reply:
Nó chỉ dùng để kiểm tra lúc nó gọi thôi, còn nội dung chuỗi s vẫn không thay đổi. Em test lại thử, anh thấy đúng mà.

Reply:
done.